packages/cashier-v2/src/lib/Transfer/hooks/useExtendedTransferAccounts.ts

```diff
@@ -49,15 +45,15 @@ const sortedCryptoDerivAccounts = (accounts: TModifiedAccounts) => {
     - sorts the mt5 accounts based on group type
     - sorts the crypto accounts alphabetically
 */
-const useExtendedTransferAccounts = (accounts: THooks.TransferAccounts) => {
-    const { data: activeAccount, isLoading: isActiveAccountLoading } = useActiveAccount();
-    const { getConfig, isLoading: isCurrencyConfigLoading } = useCurrencyConfig();
-
-    const isLoading = !accounts || isActiveAccountLoading || isCurrencyConfigLoading;
 
+const useExtendedTransferAccounts = (
+    accounts: THooks.TransferAccounts = [],
+    activeAccount: THooks.ActiveAccount,
+    getConfig: THooks.GetCurrencyConfig
```

Comment:
What is the reason behind passing getConfig as a param here instead of using useCurrencyConfig hook?